File: EquipmentApi/EquipmentApi/Dtos/ReadDtos/EquipmentPositionHistoryDto.cs
using System;

namespace EquipmentApi.Dtos.ReadDtos
{
    public class EquipmentPositionHistoryDto
    {
        public Guid EquipmentId { get; set; }
        public DateTime Date { get; set; }
        public float Lat { get; set; }
        public float Lon { get; set; }

        public EquipmentDto Equipment { get; set; }
    }
}

File: forestequipment/src/main/java/com/vitor/forestequipment/dtos/EquipmentPositionHistoryDto.java
package com.vitor.forestequipment.dtos;

import java.time.LocalDateTime;
import java.util.UUID;

public class EquipmentPositionHistoryDto {

	private UUID equipment;

	private double lat;

	private double lon;
	
	private LocalDateTime date;

	public UUID getEquipment() {
		return equipment;
	}

	public void setEquipment(UUID equipment) {
		this.equipment = equipment;
	}

	public double getLat() {
		return lat;
	}

	public void setLat(double lat) {
		this.lat = lat;
	}

	public double getLon() {
		return lon;
	}

	public void setLon(double lon) {
		this.lon = lon;
	}
	
	public LocalDateTime getDate() {
		return date;
	}
	
	public void setDate(LocalDateTime date) {
		this.date = date;
	}


}
